packages/mongo/oplog_tests.js

```diff
@@ -180,6 +182,30 @@ process.env.MONGO_OPLOG_URL && Tinytest.addAsync(
   },
 );
 
+process.env.MONGO_OPLOG_URL && Tinytest.addAsync(
+  'mongo-livedata - oplog - options.oplogExcludeCollections',
+  async test => {
+    const testCollectionName = "test-" + randomId;
+    if (!Meteor.settings.packages) Meteor.settings.packages = {};
+    if (!Meteor.settings.packages.mongo) Meteor.settings.packages.mongo = {};
+    // Meteor.settings.packages.mongo.oplogExcludeCollections = [testCollectionName];
+
+    const myOplogHandle = new MongoInternals.OplogHandle(process.env.MONGO_OPLOG_URL, testCollectionName);
+    MongoInternals.defaultRemoteCollectionDriver().mongo._setOplogHandle(myOplogHandle);
+
+    const TestCollection = new Mongo.Collection(testCollectionName);
+
+    let handle = TestCollection.find({ foo: 'bar' }).observeChanges({
+      added(id, fields) {
+        console.log('TEST added:', id, fields);
+        // Not working anymore! :(
```

Comment:
Without using my custom `OplogHandle`, this console.log works just fine! So there has to be something wrong with my implementation of setting `myOplogHandle` to be used with the `defaultRemoteCollectionDriver`.